app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -244,16 +244,32 @@ private void getDisplayModes() {
     }
 
     @TargetApi(23)
-    private Display.Mode findBestDisplayMode(Float refreshRate) {
-        if (mDisplayModes == null || refreshRate == null) return null;
-
-        int sourceRate = Math.round(refreshRate);
-        for (Display.Mode mode : mDisplayModes){
-            int rate = Math.round(mode.getRefreshRate());
-            if (rate == sourceRate || rate == sourceRate * 2) return mode;
+    private Display.Mode findBestDisplayMode(MediaStream videoStream) {
+        if (mDisplayModes == null || videoStream.getRealFrameRate() == null) return null;
+
+        int curWeight = 0;
+        Display.Mode bestMode = null;
+        int sourceRate = Math.round(videoStream.getRealFrameRate() * 100);
```

Comment:
Why multiplied by 100?

Reply:
I go to Int to avoid float comparison issues.
And I multiply by 100 to get, e.g., 23.97 (24 * 1000 / 1001, common movie frame rate)-> 2397.